platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLafOptionsPanel.java

```diff
@@ -51,9 +64,49 @@ public class FlatLafOptionsPanel extends javax.swing.JPanel {
     public FlatLafOptionsPanel(FlatLafOptionsPanelController controller) {
         this.controller = controller;
         initComponents();
+		initAccentColor();
```

Comment:
Some indent problems in this patch/file (mix of tabs and spaces). Existing indents are spaces, so maybe use spaces.